webui/react/src/components/Badge.tsx

```diff
@@ -11,10 +11,13 @@ export enum BadgeType {
   Default,
   Id,
   State,
+  Custom,
```

Comment:
**suggestion:** for this, I know I threw a wrench with a new type of Badge (dark blue). My thinking is that just adding another type would be better than opening the fb and bg colors up as props because the colors have to be hardcoded via props. Adding another type would allow us to still use the CSS modules to pull in the appropriate color from the themes.

Reply:
I added a Header type for resourcePoolCard tags in the header. 
this custom type is also used in SlotAllocationBar if we want to add a defined type for that in Badge we'd have to define and support a type like this https://github.com/determined-ai/determined/pull/1717#discussion_r543632589